oscps-lib/src/thermodynamics.rs

```diff
@@ -0,0 +1,190 @@
+//! # Thermodynamics
+//!
+//! This module will hold all the functions related to calculating themrodynamic properties for the
+//! blocks and chemical species.
+
+// use uom::si::f32::{Pressure, ThermodynamicTemperature};
+use uom::si::f64::*; 
+use uom::si::mass::kilogram;
+use uom::si::thermodynamic_temperature::kelvin;
+use uom::si::pressure::pascal;
+use crate::component::Chemical;
+
+pub enum ThermodynamicConstants {
+    UniversalGasConstant, // R
+    StandardTemperature,  // T_0
+    StandardPressure,     // P_0
+    AvogadroNumber,       // N_A
+}
+
+/// Enum for representing different types of thermodynamic constant values
+pub enum ConstantValue {
+    Pressure(Pressure),
+    Temperature(ThermodynamicTemperature),
+    Dimensionless(f64),
+}
+
+impl ThermodynamicConstants {
```

Comment:
Defining these here is fine, possibly there is a more elegant way to define constants, but I would have to research it more.